docs/features/file_formats.md

```diff
@@ -1 +1,69 @@
-# Under construction
+# Input and Output file formats
+
+The GeoModel is the Data Model powered by RINGMesh (see the full description of the GeoModel [here](/features/geomodel).
+RINGMesh proposes an home-made file format with the `.gm` extension to read and write GeoModels.
+
+Beyond that, RINGMesh is able to deal with various file format. We present here an exhaustive list of input and output file format
+supported by RINGMesh. These file format are different if you are dealing with 2D or 3D GeoModels.
+
+## Input 
+
+### GeoModel2D 
+
+| Extension          |     Software    | 
+| -------------      | -------------   |
+| .gm                | RINGMesh        |
+| .model             | Stradivarius    |
+| .svg               | Inkscape        |
+| .msh (in progress) | GMSH            |
+
+### GeoModel3D
+
+| Extension          |     Software    |
+| -------------      | -------------   |
+| .gm                | RINGMesh        |
+| .ml                | SKUA-GOCAD      |
+| .so                | SKUA-GOCAD      |
+| .msh (in progress) | GMSH            |
```

Comment:
MSH format will be done soon? If no maybe we should not mention formats that are not fully functional.

Reply:
Yes I am currently working on input and output for both 2D and 3D models. There is a PR on ringmesh repo but it is not perfectly working yet.